PWGPP/TPC/AliTPCcalibResidualPID.cxx

```diff
@@ -203,7 +204,6 @@ AliTPCcalibResidualPID::AliTPCcalibResidualPID(const char *name)
     fTree_tanTheta_vs(0.),
     fTree_distance_nb(0.),
     fTree_distance_vs(0.),
```

Comment:
Hi @MartinSchmidtTuebingen ,
it seems that you forgot to remove a comma here.

Cheers,
Diego